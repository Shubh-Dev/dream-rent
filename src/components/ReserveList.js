import React from 'react';

const ReserveList = () => (
  <div className="reserve-list-container">
    <h1>Reserve List goes here</h1>
  </div>
);

export default ReserveList;
