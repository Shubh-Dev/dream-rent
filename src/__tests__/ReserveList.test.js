import React from 'react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer';
import ReserveList from '../components/ReserveList';
import store from '../redux/configureStore';
import { Provider } from 'react-redux';

describe('ReserveList', () => {
    it('renders correctly', () => {
        const tree = TestRenderer.create(
        <Provider store={store}>
            <ReserveList />
        </Provider>,
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
    });