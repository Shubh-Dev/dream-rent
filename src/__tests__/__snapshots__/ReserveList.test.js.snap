// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReserveList renders correctly 1`] = `
<div
  className="reserve-list-container"
>
  <h1>
    Reserve List goes here
  </h1>
</div>
`;
